js/components/Leaderboard/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { map } from 'lodash';
+
+import { RestApiClient } from '@dosomething/gateway';
+import Table from '../Table';
+
+class Leaderboard extends React.Component {
+	constructor() {
+		super();
+
+		this.state = {
+			jsonReturnedValue: []
+		}
+
+		this.api = new RestApiClient('http://rogue.test/');
```

Comment:
this base url won't work in prod. Did you try grabbing the base url with `window.location`?

Reply:
actually blah `window.location` won't work because you are pointing to Rogue 🤦‍♂️ However we should use the right rogue url :)

Reply:
yes - thanks for catching! Should I write rogue-thor.dosomething.org in order to test and then if that works make another pr to update to rogue.dosomething.org? I am tempted to just do rogue.dosomething.org since no one is really using it anyway?

Reply:
yea you can just hardcode prod